### DISCUSSION TITLE
Use existing customer managed kms key

### DISCUSSION THREAD

**Discussion by arunthomas32** [getlift/lift#259] [Constructs]
Hi how can i use the existing kms key for encryption?
Currently we have only 3 options:
`encryption:null `
makes encryption disabled

`encryption: kmsManaged`
 uses the defaul aws sqs key

```
encryption:kms
encryptionKey:<keyvalue>
```

But this will create a new kms key with the keyvalue that we provide

My current scenario is that i already have a key that has been used for other encryptions as well and I need to reuse that. How can we use the existing key without creating a new one

![image](https://user-images.githubusercontent.com/102296164/188103577-7a0053cf-bcd2-44fd-a8c5-882ea0b4dd37.png)

**Comment by mnapoli** [comment]
Hi, at the moment that isn't supported by Lift unfortunately. To be honest I'm not sure if the fact that the custom key gets created is intentional actually 🤔

**Comment by aran112000** [comment]
Hi, being able to specify an ARN of a custom KMS key would be really nice.

We have a DR plan where we have cross-account backups and use KMS keys from a separate Organisation Managed Account restores can be achieved even if the AWS account was ever lost for example.
